src/css/_covid.scss

```diff
@@ -705,7 +705,8 @@ border: none;
 
 .jumbotron + .container {
   padding-top:1.5rem;
-  padding-bottom:2.5rem;
+  /* this bottom padding causes an undesirable white band on v2-state-dashboard at bottom */
```

Comment:
This change slightly squishes the bottom margin right above the footer on other pages, seems better with this padding

Reply:
OK.  Will need your help removing that white band - not sure why it only shows up on v2-state-dashboard.